srcpkgs/logisim-evolution/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'logisim-evolution'
+pkgname=logisim-evolution
+version=3.8.0
+revision=1
+hostmakedepends="openjdk17 gradle"
+depends="openjdk17-jre"
+short_desc="Digital logic design tool and simulator"
+maintainer="voidbert <humbertogilgomes@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/logisim-evolution/logisim-evolution"
+changelog="https://github.com/logisim-evolution/logisim-evolution/raw/master/CHANGES.md"
+distfiles="https://github.com/logisim-evolution/logisim-evolution/archive/refs/tags/v${version}.tar.gz"
+checksum=7a49558a490a0df886b3ea488f12e4ea887a2ec8c278872dca39b2bebfaed3ee
+
+do_build() {
+	export JAVA_HOME=/usr/lib/jvm/openjdk17
+	gradle --no-daemon shadowJar
+}
+
+do_install() {
+	vinstall build/libs/logisim-evolution-${version}-all.jar 0644 usr/lib/logisim-evolution logisim-evolution.jar
+
+	vbin ${FILESDIR}/logisim-evolution
+
+	vinstall artwork/logisim-evolution-icon.svg 0644 usr/share/icons/hicolor/scalable/apps com.github.reds.LogisimEvolution.svg
+	vinstall support/Flatpak/com.github.reds.LogisimEvolution.desktop 0644 usr/share/mime/packages
```

Comment:
```suggestion
	vinstall support/Flatpak/com.github.reds.LogisimEvolution.xml 0644 usr/share/mime/packages
```
my bad -- i typo'd

Reply:
No problem. And thanks for your patience!